src/signals/incident-management/containers/IncidentDetail/components/StatusForm/index.js

```diff
@@ -115,7 +123,7 @@ const StatusForm = ({ defaultTexts, incident, onClose }) => {
 
       form.controls.text.updateValueAndValidity();
     });
-  }, [form.controls.status.valueChanges, form.controls.text]);
+  }, [form.controls.status.valueChanges, form.controls.text, form.controls.send_email.valueChanges]);
```

Comment:
I don't see `form.controls.send_email.valueChanges` being used in the `useEffect` handler

Reply:
Correct, I removed it!